neo4j-bom/build.gradle

```
@@ -2,12 +2,14 @@ plugins {
     id("io.micronaut.build.internal.bom")
 }
 
-// disabled until catalog is updated for 4.0.x, to avoid these build errors
-//> Version catalogs are not compatible:
-//The following versions were present in the baseline version but missing from this catalog:
-//        - neo4j-compat
-//        - neo4j-driver-compat
-//The following libraries were present in the baseline version but missing from this catalog:
-//        - neo4j
-//        - neo4j-bolt
-tasks.checkVersionCatalogCompatibility { enabled = false }
+micronautBom {
+    suppressions {
+        acceptedLibraryRegressions.add("neo4j")
+        acceptedLibraryRegressions.add("neo4j-bolt")
+        acceptedLibraryRegressions.add("neo4j-compat")
+        acceptedLibraryRegressions.add("neo4j-driver-compat")
+
+        acceptedVersionRegressions.add("neo4j-compat")
+        acceptedVersionRegressions.add("neo4j-driver-compat")
```

I think we only need:

```suggestion
        acceptedLibraryRegressions.add("neo4j")
        acceptedLibraryRegressions.add("neo4j-bolt")

        acceptedVersionRegressions.add("neo4j-compat")
        acceptedVersionRegressions.add("neo4j-driver-compat")
```